code/api/src/modules/subscription/model.js

```diff
@@ -8,6 +8,9 @@ module.exports = function(sequelize, DataTypes) {
     },
     crateId: {
       type: DataTypes.INTEGER
+    },
+    nextDeliveryDate: {
+      type: DataTypes.DATE
```

Comment:
What format is DataTypes.DATE? Or does it take a variety of formats?

Reply:
Most definitely, date can be formatted in a variety of ways.

Reply:
I just tried out passing a couple of different formats to the mutation and they all worked. I tried the following:
`12-12-2020`
`Wed December 9, 2020`
`Dec 11 2020`

These all get stored in the database in the following format: `2020-12-11 00:00:00-08` Datetime with timezone which is what we're specifying with the `type: DataTypes.DATE` code above.

However, GraphQL returns a string of ["long seconds"](https://jacobruiz.com/blog/2019/6/11/why-use-custom-date-scalar-types-in-graphql) that looks like this `1607673600000` which I think is some standard date format that represents the total number of seconds elapsed since Jan 1 of some year in the 1900s. This is happening because GraphQL does not have a date data type out of the box. It's something that we would need to define or install a package for it! 

The [graphql-iso-date](https://www.npmjs.com/package/graphql-iso-date) package looks promising. This is something we can definitely look into to make FE life easier.